server/models/User.js

```diff
@@ -19,6 +20,10 @@ const userSchema = new mongoose.Schema({
   register_date: {
     type: Date,
     default: Date.now
+  },
+  profileId: {
+    type: Schema.Types.ObjectId,
+    ref: "Profile"
```

Comment:
```suggestion
    ref: "Profile",
    required: true
```
Please make this a required attribute

Reply:
OK